package.json

```diff
@@ -22,6 +22,7 @@
         "@mdx-js/mdx": "^1.6.19",
         "@mdx-js/react": "^1.6.19",
         "antd": "^3.23.2",
+        "cross-env": "^7.0.3",
```

Comment:
This looks like a devDependency actually. Though I also think we don't really support development on Windows, so not sure how much is this needed.

Reply:
This PR is based on [this Netlify tutorial](https://www.netlify.com/blog/2020/04/23/enable-gatsby-incremental-builds-on-netlify/?utm_source=blog&utm_medium=gatsby-perf-jl&utm_campaign=devex) but yeah we don't necessarily need it. Also, Netlify calls `gatsby build` and I believe it doesn't install dev dependencies, hence this is here

Reply:
Thinking again maybe we should keep it in case a contributor uses Windows. Not sure if they'd currently run into other problems though.